eZ/Publish/Core/Repository/UserService.php

```diff
@@ -1135,6 +1160,9 @@ protected function createPasswordHash($login, $password, $site, $type)
             case User::PASSWORD_HASH_PLAINTEXT:
                 return $password;
 
+            case User::PASSWORD_BCRYPT:
+                return password_hash($password, PASSWORD_BCRYPT);
```

Comment:
I've changed it to 255 after a talk with @andrerom, this will allow support for future formats/encryptions